### PR TITLE
chore: sentry tracing report disabled by default in dev mode

### LINKED ISSUE #328: Make sentry not run by default in dev mode
When we are developing the app, we should not be sending events to sentry by default when we are in development mode.

### PR DESCRIPTION
**Pull request by tomaspalma**
Closes #328 

This takes care of not sending tracing events to sentry when in development.

### CODE REVIEW

**Review by thePeras — CHANGES_REQUESTED** [review]
I think you can use just the `enabled` property.
Read more about [here](https://gist.github.com/MirzaLeka/d709d0925d9b749d1b9068abf4b5ad50)

**Comment by tomaspalma** [comment]
I understand what you're saying but I was not trying to disable it entirely.

The goal was to allow to disable specific features.

This way I can seamlessly test the bug report without sending tracing reports to sentry.

**Comment by thePeras** [comment]
> I understand what you're saying but I was not trying to disable it entirely.
> 
> The goal was to allow to disable specific features.
> 
> This way I can seamlessly test the bug report without sending tracing reports to sentry.

Ok, I see

File: .env.example [hunk 1]
```diff
@@ -7,5 +7,7 @@ VITE_APP_SEMESTER=
 VITE_APP_SITE_TITLE=
 VITE_APP_BACKEND_URL=
 
+VITE_APP_SENTRY_DSN=https://01f0882e6aa029a125426e4ad32e6c18@o553498.ingest.us.sentry.io/4507775325437952
+VITE_APP_SENTRY_TRACING=0
 # To upload source maps to Sentry (Optional)
-SENTRY_AUTH_TOKEN=
```
Comment (thePeras) [inline .env.example:11]: I think there is no need for `VITE` prefix here as this env variable is not used in the React app, but in the cli or GitHub CI workflow.

**Pull request merged**